tests/test_runner/test_parallel.py

```diff
@@ -52,6 +53,39 @@ def dummy_test(self):
 
 class RemoteTestResultTest(SimpleTestCase):
 
+    def test_was_successful_no_events(self):
+        result = RemoteTestResult()
+        self.assertTrue(result.wasSuccessful())
+
+    def test_was_successful_one_success(self):
+        result = RemoteTestResult()
+        result.addSuccess(None)
+        self.assertTrue(result.wasSuccessful())
+
+    def test_was_successful_one_expected_failure(self):
+        result = RemoteTestResult()
+        result.addExpectedFailure(None, ValueError("woops"))
+        self.assertTrue(result.wasSuccessful())
+
+    def test_was_successful_one_skip(self):
+        result = RemoteTestResult()
+        result.addSkip(None, "Skipped")
+        self.assertTrue(result.wasSuccessful())
+
+    def test_was_successful_one_error(self):
+        result = RemoteTestResult()
+        result.addError(None, ValueError("woops"))
+        self.assertFalse(result.wasSuccessful())
+
+    def test_was_successful_one_failure(self):
+        result = RemoteTestResult()
+        result.addFailure(None, ValueError("woops"))
+        self.assertFalse(result.wasSuccessful())
+
+    def test_picklable(self):
+        result = RemoteTestResult()
+        pickle.loads(pickle.dumps(result))
+
```

Comment:
This works without a patch, so we can move it to a separate commit.

Reply:
Done

Reply:
I think you meant just the picklable test here, since the other tests cover functionality added later when changing the inheritance of `RemoteTestResult`, right?

Reply:
Yes.